front/src/reducers/osrdconf/operationalStudiesConf/simulationConfReducers.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, expect } from 'vitest';
+import { createStoreWithoutMiddleware } from 'Store';
+
+import type { OsrdConfState } from 'applications/operationalStudies/consts';
+
+import { infraState } from 'reducers/infra';
+import { defaultCommonConf } from 'reducers/osrdconf/osrdConfCommon';
+import testCommonConfReducers from 'reducers/osrdconf/osrdConfCommon/__tests__/utils';
+import {
+  operationalStudiesConfInitialState,
+  operationalStudiesConfSlice,
+} from 'reducers/osrdconf/operationalStudiesConf';
+
+const createStore = (initialStateExtra?: Partial<OsrdConfState>) =>
```

Comment:
For this file, the function is used only once, and without any params, so you can simply remove it for this time
```suggestion
const createStore = () =>
```

Reply:
Done